tests/Unit/Http/Controllers/PartnerControllerTest.php

```diff
@@ -0,0 +1,138 @@
+<?php
```

Comment:
em test những testcase gì trong file này?

Reply:
- trong file này em test:
-- function testIndex(): Có một testCase là tìm kiếm khách sạn của partner
-- function testCreate: Có 1 testCase  là lấy tất cả provinces
-- fucntion testStore: Có 1 testCase thêm khách sạn thành công
-- function testUpload: Approved thành công
-- function testDeny: Denied thành công
-- function testCheckOut: Thanh toán thành công